visualization/app/codeCharta/state/store/metricData/nodeMetricData/nodeMetricData.service.ts

```diff
@@ -57,6 +57,11 @@ export class NodeMetricDataService implements StoreSubscriber, FilesSelectionSub
 		return metric?.maxValue
 	}
 
+	getMinMetricByMetricName(metricName: string) {
```

Comment:
```suggestion
	getMinValueOfMetric(metricName: string) {
```

Reply:
This was named similar to line 55, maybe changing both lines is the better sollution?

Reply:
Yes, go for it. 👍